components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/scope/AbstractRoleBasedScopeIssuer.java

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License header files need to be updated.
Please refer to "Re: License Header for WSO2 code"

Reply:
fixed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2084/commits/7520bd12a17ebf1cd3bcd9b9941c35dae1675957